.gitmodules

```diff
@@ -1,9 +1,3 @@
-[submodule "lib/forge-std"]
-	path = lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
```

Comment:
Do you think it might be a good idea if we keep this `forge-std` dependency?

1/ what we are doing means that if we somehow need to update `forge-std` to get newer feature, we are forced to rely on dependency -- where we might not want to touch `v4-core` and `v4-periphery` since the code has `frozen` 

2/ and so far i think `forge-std` have not caused us any conflicts yet 

3/ we do this for oz to prevent conflicts due to breaking changes

Reply:
same for `forge-gas-snapshot` probably